<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainScreen</class>
 <widget class="QWidget" name="MainScreen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1151</width>
    <height>707</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>540</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QWidget" name="widgetTitle" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="welcomeLabel">
        <property name="font">
         <font>
          <pointsize>20</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Welcome</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="userNameTitle"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QWidget" name="widget_2" native="true">
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QLabel" name="label">
           <property name="font">
            <font>
             <pointsize>15</pointsize>
            </font>
           </property>
           <property name="text">
            <string>List rented book:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTableView" name="bookTableView"/>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="buttonWidget" native="true">
        <layout class="QVBoxLayout" name="verticalLayout">
         <item alignment="Qt::AlignHCenter">
          <widget class="QPushButton" name="addBookBtn">
           <property name="minimumSize">
            <size>
             <width>90</width>
             <height>60</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>90</width>
             <height>60</height>
            </size>
           </property>
           <property name="text">
            <string>Rent Book</string>
           </property>
          </widget>
         </item>
         <item alignment="Qt::AlignHCenter">
          <widget class="QPushButton" name="returnBookBtn">
           <property name="minimumSize">
            <size>
             <width>90</width>
             <height>60</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>90</width>
             <height>60</height>
            </size>
           </property>
           <property name="text">
            <string>Return Book</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item alignment="Qt::AlignRight">
    <widget class="clickableLabel" name="goBack">
     <property name="font">
      <font>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="frameShape">
      <enum>QFrame::Panel</enum>
     </property>
     <property name="text">
      <string>Go back to Log In</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>clickableLabel</class>
   <extends>QLabel</extends>
   <header location="global">clickableLabel.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
